CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+### v0.36.1
+
+Fixes issue with the CID log which in certain scenarios caused Webnative to load the wrong version of an account's file system.
```

Comment:
Small grammar nitpick, but entirely necessary maybe since it's just the changelog
```suggestion
Fixes an issue with the CID log, which, in certain scenarios, caused Webnative to load the wrong version of an account's file system.
```